packages/core/src/components/document-component-demo/document-component.demo.tsx

```diff
@@ -286,55 +318,61 @@ export class DocumentComponentDemo implements ComponentInterface {
     const embeddedDocuments = this.getEmbeddedDocumentComponents(documentComponent);
 
     return (
-      <dso-document-component
-        annotated={this.isAnnotated(documentComponent)}
-        bevatOntwerpInformatie={!!documentComponent.bevatOntwerpInformatie}
-        filtered={
-          this.isOpen(documentComponent)
-            ? this.isFiltered(documentComponent)
-            : this.hasFilteredChildren(documentComponent)
-        }
-        genesteOntwerpInformatie={this.hasNestedDraft(documentComponent)}
-        gereserveerd={documentComponent.gereserveerd}
-        heading="h2"
-        inhoud={documentComponent.inhoud}
-        label={documentComponent.labelXml}
-        openAnnotation={this.isOpenedAnnotation(documentComponent)}
-        notApplicable={this.isNotApplicable(documentComponent) || path.some((p) => this.isNotApplicable(p))}
-        nummer={documentComponent.nummerXml}
-        onDsoAnnotationToggle={() => this.handleAnnotationToggle(documentComponent)}
-        onDsoOpenToggle={() => this.handleOpenToggle(documentComponent)}
-        onDsoOzonContentAnchorClick={(e) => this.handleOzonContentAnchorClick(e)}
-        open={this.isOpen(documentComponent)}
-        opschrift={documentComponent.opschrift}
-        type={documentComponent.type}
-        vervallen={documentComponent.vervallen}
-        wijzigactie={documentComponent.wijzigactie}
-        recursiveToggle={this.recursiveToggleState(documentComponent)}
-        onDsoRecursiveToggle={(e) => this.handleRecursiveToggle(documentComponent, e.detail)}
-      >
-        {this.isOpenedAnnotation(documentComponent) && (
-          <div slot="annotations">
-            <dso-annotation-locatie
-              active={this.isCheckedSlideToggle(documentComponent)}
-              gewijzigde-locatie
-              locatieNoemer={"Winkelgebied"}
-              onDsoActiveChange={() => this.handleSelectableChange(documentComponent)}
-            >
-              <span class="symboolcode" slot="symbool" data-symboolcode="vszt030"></span>
-            </dso-annotation-locatie>
-          </div>
-        )}
-        {this.showContent(documentComponent) && embeddedDocuments?.documentComponents.length && (
-          <ul class="dso-document-component-list">
-            {embeddedDocuments.documentComponents.map((d) => (
-              <li key={d.documentTechnischId}>
-                <DocumentComponent path={[...path, d]} />
-              </li>
-            ))}
-          </ul>
-        )}
-      </dso-document-component>
+      (this.mode === "document" ||
+        (this.mode === "table-of-contents" && this.showInTableOfContents(documentComponent.type))) && (
```

Comment:
Ik kan moeilijk zien of er nog meer changes zijn, kan je uitleggen waar dit goed voor is?

Reply:
Leden worden niet in de inhoudsopgave getoond en dus heb ik de check omgedraaid sluit ik nu type "LID" uit, waardoor ook in 1x "DIVISIE" en "DIVISIETEKST" in de inhoudsopgave getoond worden. Zeg maar 2 vliegen in klap.